recipes/rad_haplotyper/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: rad_haplotyper
+  version: '1.1.6'
+
+source:
+  fn: v1.1.6.tar.gz
+  url: https://github.com/chollenbeck/rad_haplotyper/archive/v1.1.6.tar.gz
+  sha256: da7f663922ca66f4b736775df16e70c423dad74aeb03bc4d732572ff01d8466d
+
+build:
+  number: 0
+  skip: True # [osx]
```

Comment:
Please try building on OSX as well or leave a comment here, explaining why OSX won't comply

Reply:
This is a post-analysis script to use with the dDocent pipeline which currently has dependencies that do not compile on OS X.  I'm trying to avoid any confusion.